src/calculateRentalCost.js

```diff
@@ -28,7 +28,19 @@
  */
 
 function calculateRentalCost(days) {
-  // write code here
+  const dayPrice = 40;
+  const bigDiscount = 50;
+  const smallDiscount = 20;
+
+  let total = days * dayPrice;
+
+  if (days >= 7) {
+    total -= bigDiscount;
+  } else if (days >= 3) {
+    total -= smallDiscount;
+  }
```

Comment:
Avoid `else if `when it is not necessary. 
And you can return the value right away after you've found the right condition

```suggestion
  if (days >= 7) {
    return total - bigDiscount;
  }  
  
  if (days >= 3) {
    return total - smallDiscount;
  }
```